orttraining/orttraining/python/training/pytorch_trainer_options.py

```diff
@@ -0,0 +1,417 @@
+import cerberus
+import torch
+
+from .optim import lr_scheduler
+from .amp import loss_scaler
+
+
+class PytorchTrainerOptions(object):
+    r"""Settings used by ONNX Runtime training backend
+
+    The parameters are hierarchically organized to facilitate configuration through semantic groups
+    that encompasses features, such as distributed training, etc.
+
+    Although this class uses a dictionary for initialization, input validation
```

Comment:
Minor suggestion: This sentence sounds a bit like an explanation of design. Can we rephrase it for the doc here to something declarative like, "Input validation is performed on the input dictionary during instantiation to ensure that supported parameters and values are passed in."